web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
+            <li><a href="{{ pathPrefix }}/config">Config</a></li>
```

Comment:
At what point do we stop adding things to the navbar? I expect to use rules (once it's properly fleshed out), config, storage and flags more often than targets. I'm a bit surprised targets is being used this often, do your dashboards not allow you to find down instances easily?

I don't think the targets as we currently have them is something we'll be able to keep around, as the format isn't suitable once you get to large setups with thousands of targets. The way I'd like to see things is that we'd have a list of jobs on the Status page, which then link down to the targets for that job. Similarly for rule groups.
Having a top navbar for just a list of jobs would be odd, but having the jobs/rule groups on the status page (likely 10s of items each in a large setup) with other information sounds doable.